app/views/pieces/index.html.haml

```diff
@@ -1,9 +1,19 @@
-%ul
-  - @pieces.each do |piece|
-    .piece
-      %p.piece-title= link_to piece.title, piece_path(piece)
-      %p.piece-composer= piece.composer
-
 %p= link_to "Add New Piece", new_piece_path
 
-= paginate @pieces
+=form_tag(pieces_path, :method => "get", id: "search-form") do
+  =text_field_tag :title_search, params[:title_search], placeholder: "Search by Title"
+  =submit_tag "Title Search", :name => nil, label: "Title Search", id: "title_search_button"
```

Comment:
Line is too long. [92/80]<br>The = symbol should have one space separating it from code